core-blocks/image/edit.js

```diff
@@ -344,6 +344,7 @@ class ImageEdit extends Component {
 							const ratio = imageWidth / imageHeight;
 							const minWidth = imageWidth < imageHeight ? MIN_SIZE : MIN_SIZE * ratio;
 							const minHeight = imageHeight < imageWidth ? MIN_SIZE : MIN_SIZE / ratio;
+							const isRLT = window.getComputedStyle( document.querySelector( '.editor-block-list__layout' ) ).direction === 'rtl';
```

Comment:
Should not be getting state from DOM. Can use selector for it within withSelect:

```js
isRTL: select( 'core/editor' ).getEditorSettings().isRTL,
```

Then access as `prop.isRTL`.

Reply:
Right, was looking at that. RTL is a bit of a special case. `getComputedStyle()` has the advantage of catching both the `dir` attribute and CSS `direction`, and dynamically gets the proper state used at the moment. 

The `dir` attribute and CSS `direction` usually are set the same as the editor setting but in some cases may be reversed. For example the `code` block in RTL languages should still be LTR. I agree this generally wouldn't apply to image blocks, so using the editor setting should be sufficient.

Also it seems we don't yet have good RTL support. The classic editor gets an additional button in RTL languages that lets the user reverse text direction. We'll probably need to add that in most blocks with `RichText`, and set the `code` block to LTR by default.